core/src/main/scala/akka/persistence/r2dbc/ConnectionFactoryProvider.scala

```diff
@@ -95,6 +95,8 @@ class ConnectionFactoryProvider(system: ActorSystem[_]) extends Extension {
       .maxAcquireTime(JDuration.ofMillis(settings.acquireTimeout.toMillis))
       .acquireRetry(settings.acquireRetry)
       .maxIdleTime(JDuration.ofMillis(settings.maxIdleTime.toMillis))
+      // setting lifetime due to issue https://github.com/r2dbc/r2dbc-pool/issues/129
+      .maxLifeTime(JDuration.ofDays(365 * 100))
```

Comment:
workaround for https://github.com/r2dbc/r2dbc-pool/issues/129
found this when verifying performance, connections are closed immediately otherwise